doc/rst/platforms/infiniband.rst

```diff
@@ -105,24 +105,33 @@ Selecting a Spawner
 Under the covers, ``gasnetrun_ibv``-based launchers must figure out
 how to spawn jobs and get them up and running on the compute nodes.
 GASNet's three ways of doing this on InfiniBand systems are ``ssh``,
-``pmi``, and ``mpi``.  When MPI is detected at configure time, it will
-be selected as the default, but we recommend using one of the other
-options if possible.
-
-* ``ssh``: Based on our experience, we recommend launching with the
-  ``ssh`` option, if possible.  This requires the ability to ``ssh``
-  to the system's compute nodes, which is not supported by all
-  systems, depending on how they are configured.  See the following
-  sub-section for details on this option.
+``pmi``, and ``mpi``, described just below.  When MPI is detected at
+configure time, it will be selected as the default, but we recommend
+using one of the other options if possible.  This can be done by
+setting the ``GASNET_IBV_SPAWNER`` environment variable, whose legal
+values are:
+
+* ``ssh``: Based on our experience, this is the preferred option, when
+  possible.  This requires the ability to ``ssh`` to the system's
+  compute nodes, which is not supported by all systems, depending on
+  how they are configured.  See the following sub-section for details
+  on this option.
   
 * ``pmi``: When GASNet's configure step detects a PMI-capable job
   scheduler like Slurm, ``pmi`` can be the next best choice because it
-  often "just works" and can reduce overhead compared to ``mpi``.
-
+  often "just works" and can reduce overhead compared to ``mpi``.  For
+  more information about this option, including how to configure job
+  launch via ``PMIRUN_CMD``, see the `GASNet README for the PMI-based
+  spawner
+  <https://bitbucket.org/berkeleylab/gasnet/src/master/other/pmi-spawner/README>`_
+  (also available at
+  ``$CHPL_HOME/third-party/gasnet/gasnet-src/other/pmi-spawner/README``).
+  
 * ``mpi``: When the previous cases are not options, ``mpi`` serves as
-  a reasonable last resort.  Note that it can incur a performance
-  penalty because MPI will be running concurrently with GASNet.  See
-  the second subsection below for tips on this option.
+  a reasonable last resort.  Note that it may, depending on its
+  configuration, incur a performance penalty due to competition
+  between MPI and GASNet for limited communication resources.  See the
+  second subsection below for best practices when using this option.
```

Comment:
i think the content here is fine, but rather than relying on the reader to find the right section I think this should just link directly to that section (even just link to the MPI section). not at show stopper, but a mild improvement

Reply:
You just want to see if I remember how to create rst hyperlinks, or even that it's possible, don't you?  Note that these are a pre-existing condition independent of this PR, but I've added them while here.  Thanks!